src/ingredient.js

```diff
@@ -0,0 +1,16 @@
+class Ingredient {
+  constructor(id, name, estimatedCost) {
+    this.id = id;
+    this.name = name;
+    this.estimatedCostInCents = estimatedCost
+  }
+
+  calculateIngredientCost() {
+    let ingredientCost = this.estimatedCostInCents / 100;
```

Comment:
should this be a const?

Reply:
Good catch, your right, I'll fix that and push it up!